DFC.Digital/DFC.Digital.Services.SendGrid.UnitTests/packages.config

```diff
@@ -0,0 +1,25 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<packages>
+  <package id="FakeItEasy" version="4.5.1" targetFramework="net472" />
+  <package id="FluentAssertions" version="5.6.0" targetFramework="net472" />
+  <package id="MSTest.TestAdapter" version="1.2.1" targetFramework="net472" />
+  <package id="MSTest.TestFramework" version="1.3.2" targetFramework="net472" />
+  <package id="Newtonsoft.Json" version="9.0.1" targetFramework="net472" />
+  <package id="Sendgrid" version="9.10.0" targetFramework="net472" />
+  <package id="StyleCop.Analyzers" version="1.0.2" targetFramework="net472" developmentDependency="true" />
+  <package id="System.IO" version="4.3.0" targetFramework="net472" />
+  <package id="System.Net.Http" version="4.3.3" targetFramework="net472" />
```

Comment:
This always causes issues, confirm we need this and if we do then try to keep it same as what we have in solution `4.3.4`